sunpy/net/hek2vso/hek2vso.py

```diff
@@ -150,9 +145,9 @@ def full_query(self, client_query, limit=None, progress=False):
         self.hek_results = self.hek_client.search(*client_query)
         self._quick_clean()
         return self.translate_and_query(self.hek_results,
-                                        limit=limit, progress=progress)
+                                        limit=limit)
 
-    def translate_and_query(self, hek_results, limit=None, progress=False, vso_response_format="table"):
+    def translate_and_query(self, hek_results, limit=None):
```

Comment:
Shouldn't this be deprecated instead of removed?

Reply:
Ah woops, will fix.